cluster/grpc_rpc_client.go

```diff
@@ -75,6 +75,11 @@ func NewGRPCClient(
 	return gs, nil
 }
 
+type grpcServer struct {
```

Comment:
nit: doesn't make sense to call this struct grpcClient? and also the "srv" attribute is actually a client, sounds kinda strange...

Reply:
I thought about it, my line of thought was that this struct contained information about a grpc server. But thinking now I tend to agree with you, I will change grpcServer to grpcClient and srv to cli

Reply:
We already have `GRPCClient` in this file. I would recommend the name `grpcConn`